corehq/apps/hqwebapp/views.py

```diff
@@ -864,7 +864,7 @@ def post(self, request, *args, **kwargs):
             post_params=request.POST,
             couch_user=request.couch_user,
             uploaded_file=request.FILES.get('feature_request'),
-            to_email=settings.SOLUTIONS_EMAIL,
+            to_email=settings.INTERNAL_FEEDBACK_EMAIL,
```

Comment:
that might have been a better name actually.
https://github.com/dimagi/commcare-cloud/pull/6211#discussion_r1465922233